user_flair/hero_flair.scss

```diff
@@ -626,4 +626,12 @@
 
 .flair-master-probius::before {
   background-position: 40px -2880px
+}
+
+.flair-genji::before {
+  background-position: 0 -2900px
+}
+
+.flair-master-genji::before {
```

Comment:
I won't add it in the Edit Flair, so people can't select one yet anyway. I can remove the lines, but I'll probably add them back in in a few days anyway.